example_site/src/components/ConfigurationControls.js

```diff
@@ -49,6 +49,132 @@ export default function ConfigurationControls({ isOpen }) {
     shouldShowConfirmCompleteVideoModal,
   } = watchFields;
 
+  function renderUnsavedChanges() {
```

Comment:
I think a better name for this function would be `renderUnsavedChangesModalForm`. 

There is definitely an art to variable naming, it's often touted as one of the toughest things to do! [This article](https://towardsdatascience.com/data-scientists-your-variable-names-are-awful-heres-how-to-fix-them-89053d2855be) is focusing on data science/python, but I thought it had some really nice general rules. My favorite bit is, "_Your code will be read more times than it is written._".  

When I read the original name, `renderUnsavedChanges`, I think there are "unsaved changes" that need to be rendered. If you add `ModalForm` to the end, it's a little more descriptive about what this function actually does.